sensors/cmd/Dockerfile

```diff
@@ -1,13 +1,21 @@
 FROM centos:7
+
+ARG ARGO_VERSION=v2.8.2
+
 RUN yum -y install ca-certificates openssh openssh-server openssh-clients openssl-libs curl git
 
 # OpenFass CLI
+# TODO: donwload faas instead of copying it from the svc, same as with argo-cli
 COPY assets/faas-cli /usr/local/bin/faas
 
 # Argo Workflow CLI
-COPY assets/argo-linux-amd64 /usr/local/bin/argo
+RUN curl -sSL -o /usr/local/bin/argo \
+    https://github.com/argoproj/argo/releases/download/${ARGO_VERSION}/argo-linux-amd64 \
+ && chmod +x /usr/local/bin/argo
 
-RUN argo version
+# added `|| true` to ignore "invalid configuration: no configuration has been provided"
+# TODO: argo version shouldn't rely on a cluster connection to execute
+RUN argo version || true
```

Comment:
I believe originally `argo version` here was used to check if `argo` binary was existing, adding `|| true` makes it meaningless. Since there's an unexpected error with latest `argo` cli, I suggest to remove this line for now, meanwhile file a bug in argo workflow community.

Reply:
It still outputs as part of the build the version of argo used so we may wanna keep it for that, but if you still prefer it i'll comment it out..